eyecite/tokenizers.py

```diff
@@ -336,12 +336,59 @@ class Tokenizer:
     def tokenize(self, text: str) -> tuple[Tokens, list[tuple[int, Token]]]:
         """Tokenize text and return list of all tokens, followed by list of
         just non-string tokens along with their positions in the first list."""
+
+        def _is_short_cite_following_line_number(
+            last_token: Token | None, token: Token
+        ) -> bool:
+            """
+            When we have a short cite following a line number, it can result in two
+            CitationTokens.
+            e.g., "21 Taylor, 281 F.3d at 830" will result in:
+            1) "21 Taylor, 281"
+            2) "281 F.3d at 830"
+
+            When this occurs, we want to take the latter. However we'll try to be
+            conservative and only take the second token if:
+            - last_token.volume < 50 (line numbers rarely go past 50)
+            - last_token.page == token.volume
```

Comment:
this is certainly an interesting strategy, I understand now that why you said volume too large.  I think the appropriate way to handle page numbers is not to change how we process citations but improve how we clean the text.  Perhaps a flag for cleaning page line numbers would be more effective approach.  


If I was choosing between overlapping citations 

"21 Taylor, 281"
and 
"281 F.3d at 830" 

I would try to use the one that is common.